node/service/src/overseer.rs

```diff
@@ -106,165 +109,6 @@ where
 	pub dispute_coordinator_config: DisputeCoordinatorConfig,
 }
 
-/// Create a default, unaltered set of subsystems.
-///
-/// A convenience for usage with malus, to avoid
-/// repetitive code across multiple behavior strain implementations.
-pub fn create_default_subsystems<'a, Spawner, RuntimeClient>(
```

Comment:
Was this moved into `malus`?

Reply:
There are no more `AllSubsystems` instances, so `fn prepared_overseer_builder` takes its place instead.

Reply:
I see, `malus` is now using `prepared_overseer_builder`.

Reply:
And `fn dummy_overseer_builder` respectively.